tests/Bridge/Doctrine/MongoDbOdm/CollectionDataProviderTest.php

```diff
@@ -28,6 +28,7 @@
 
 /**
  * @author Alan Poulain <contact@alanpoulain.eu>
+ * @group mongodb
```

Comment:
phpdoc and non-phpdoc annotations should not be grouped together (add a blank line). And probably the `@group` is much more important than the `@author` so it could go first.

Reply:
Cleaned up